contributors/devel/e2e-tests.md

```diff
@@ -432,7 +431,7 @@ at a custom host directly:
 export KUBECONFIG=/path/to/kubeconfig
 export KUBE_MASTER_IP="127.0.0.1:<PORT>"
 export KUBE_MASTER=local
-go run hack/e2e.go -- -v --test
+go run hack/e2e.go -- -v --test --provider=local
```

Comment:
What is the default value for `--provider`? If it is `local`, we may not need the flag.